pkg/objstore/s3/s3_test.go

```diff
@@ -26,19 +27,23 @@ http_config:
 	if cfg.Insecure != false {
 		t.Errorf("parsing of insecure failed: got %v, expected %v", cfg.Insecure, false)
 	}
-
+	if cfg.HTTPConfig.InsecureSkipVerify != true {
+		t.Errorf("parsing of insecure_skip_verify failed: got %v, expected %v", cfg.HTTPConfig.InsecureSkipVerify, false)
+	}
 }
 
 func TestValidate_OK(t *testing.T) {
 	input := []byte(`bucket: "bucket-name"
 endpoint: "s3-endpoint"
 access_key: "access_key"
 insecure: false
+insecure_skip_verify: false
```

Comment:
This should be removed right?

Reply:
whoops, yes it should. I removed this and all other occurrences of this. sorry about that. lmk if I missed any